test/integration/IncomeSimulator.t.sol

```diff
@@ -152,7 +152,7 @@ contract IncomeSimulator is BaseIntegrationTest, LogUtil {
             _inflectionPrice.toString()
         );
         string memory results =
-            _contactString(", ReachTargetInDays: ", reachTargetInDays.toString(), ", Income: ", income.toString());
+                        _contactString(", ReachTargetInDays: ", reachTargetInDays.toString(), ", Income: ", income.toString());
```

Comment:
For readability, it will be better with this log change:
Income(ether): `income.toString() / 1 ether`

Reply:
updated